src/app-layout/visual-refresh/context.tsx

```diff
@@ -417,9 +422,25 @@ export const AppLayoutInternalsProvider = React.forwardRef(
       changeHandler: 'onChange',
     });
 
-    const activeDrawerWidth = 290;
+    const [drawersMaxWidth, setDrawersMaxWidth] = useState(toolsWidth);
 
-    const { refs: drawersRefs, setFocus: focusDrawersButtons } = useFocusControl(activeDrawerId);
+    const activeDrawer = drawers?.items.find((drawer: any) => drawer.id === activeDrawerId);
```

Comment:
Can we remove (or replace) the `any` here? We should be able to update [line 417 (`const drawers = ...`)](https://github.com/cloudscape-design/components/blob/5e16764b4613b934a2d00ec37f7b2bdba54c4415/src/app-layout/visual-refresh/context.tsx#L417) similar to how we have it in the classic AppLayout: https://github.com/cloudscape-design/components/blob/d5aa98e030232223a427f2f927e4ac4b91d6c162/src/app-layout/index.tsx#L132 Typescript should then infer the types in this file correctly